app/services/symphony_client.rb

```diff
@@ -32,12 +32,9 @@ def login(user_id, password)
   end
 
   def authenticate(user)
-    request('/user/patron/authenticate',
-            headers: { 'x-sirs-sessionToken': user.session_token }, method: :post,
-            json: {
-              barcode: user.username,
-              password: user.patron_key
-            })
+    authenticated_request("/user/patron/key/#{user.patron_key}",
```

Comment:
Looks great, this is a nice solution.

I would only recommend changing the method name from `def authenticate` to `def ping` or something like that. `def session_alive?` maybe? That would mean converting return values to true/false (200 = true, anything else is false). Just some ideas.

Reply:
I think adding a one line comment to the method might be good too.

Reply:
> Looks great, this is a nice solution.
> 
> I would only recommend changing the method name from `def authenticate` to `def ping` or something like that. `def session_alive?` maybe? That would mean converting return values to true/false (200 = true, anything else is false). Just some ideas.

I like `ping` 👍 will make the change and the return values change